UnityProject/Assets/Scripts/Mining/SatchelBehaviour.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SatchelBehaviour : MonoBehaviour, IServerInventoryMove
+{
+
+	public HashSet<NamedSlot> CompatibleSlots = new HashSet<NamedSlot>() {
+		NamedSlot.leftHand,
+		NamedSlot.rightHand,
+		NamedSlot.suitStorage,
+		NamedSlot.belt,
+	};
+
+
+	public PlayerScript Player;
+	public ItemStorage Storage;
+
	void Awake() {
```

Comment:
You have some windows line endings in this PR, try changing your git config with `git config --global core.autocrlf input` so git will automatically convert all of them to LF

Reply:
Luckily I can use my IDE Automatic format And that gets rid of the strange line endings

Reply:
nope It doesn't rip

Reply:
Your editor probably has a setting for line endings too somewhere

Reply:
yeah It does it 99% of the time except for these random ones that sneak in somehow, Probably me copying back and forth from notepad